RoeiJeRot.Logic/Helper/ReservationConstraints.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using RoeiJeRot.Logic.Services;
+using static Innovative.SolarCalculator.SolarTimes;
+
+namespace RoeiJeRot.Logic.Helper
+{
+    public class ReservationConstraintsMsg
+    {
+        public bool IsValid { get; private set; }
+        public string Reason { get; private set; }
+
+        public ReservationConstraintsMsg(bool isValid, string reason)
+        {
+            IsValid = isValid;
+            Reason = reason;
+        }
+    }
+
+    public static class ReservationConstraints
+    {
+        public static ReservationConstraintsMsg IsValid(DateTime date, TimeSpan duration, IReservationService reservationService, int accId)
+        {
```

Comment:
In plaats dat je elke keer een nieuwe instantie returned kan je ook een string maken daar de message in returnen dan op het eind een instantie aanmaken.